package.json

```diff
@@ -3,7 +3,7 @@
   "version": "4.2.0",
   "description": "The official MongoDB connector for the LoopBack framework.",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```

Comment:
Bumping up the minimum supported Node.js version is often considered as a breaking change requiring semver-major release. What is our current position?